easybuild/easyconfigs/y/Yade/Yade-1.20.0-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,74 @@
+easyblock = 'CMakeMake'
+
+name = 'Yade'
+version = '1.20.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://yade-dem.org/doc/'
+description = """ Yade is an extensible open-source framework for discrete numerical models, 
+ focused on Discrete Element Method. The computation parts are written in c++ using flexible object model, 
+ allowing independent implementation of new alogrithms and interfaces. 
+ Python is used for rapid and concise scene construction, simulation control, postprocessing and debugging. """
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://launchpad.net/yade/trunk/yade-1.00.0/+download/']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('Loki', '0.1.7'),
+    ('Boost', '1.60.0', versionsuffix),
+    ('GL2PS', '1.3.9'),
+    ('Python', '2.7.11'),    
+    ('IPython', '3.2.3', versionsuffix),
+    ('PyGTS', '0.3.1', versionsuffix),
+    ('Eigen', '3.2.7'),
+    ('minieigen', '0.5.3', versionsuffix),
+    ('Qt', '4.8.7'),
```

Comment:
this depends on GLib 2.47.5, we'll need to make a variant that uses GLib 2.48.0 to be compatible with *GTS and CGAL 4.8...